projects/kit/components/input-phone-international/const/countries.ts

```diff
@@ -239,6 +239,7 @@ export const COUNTRIES: Record<string, Country> = {
     [TuiCountryIsoCode.ZA]: {mask: '+27##-###-####', name: 'Южно-Африканская Респ.'},
     [TuiCountryIsoCode.ZM]: {mask: '+260##-###-####', name: 'Замбия'},
     [TuiCountryIsoCode.ZW]: {mask: '+263#-######', name: 'Зимбабве'},
+    [TuiCountryIsoCode.RK]: {mask: '+383##-###-###', name: 'Республика Косово'},
```

Comment:
These should probably go in alphabetical order, it would be easier for us to find later and `object.keys` could be used somewhere.

Reply:
fixed